openmls/src/group/mls_group/membership.rs

```diff
@@ -259,6 +259,20 @@ impl MlsGroup {
             .collect())
     }
 
+    /// Get the [`KeyPackage`] of a member corresponding to the given
+    /// [`KeyPackageRef`]. Returns `None` if no matching [`KeyPackage`] can be
```

Comment:
But it's also returning an error. Either describe what that is, or preferably only return an `Option`.

Reply:
Good point, that should have been in the docs. I changed the function, though to return `None` when an error is bubbled up. (See my reply [here](https://github.com/openmls/openmls/pull/792#discussion_r804456019).)